### DISCUSSION TITLE
Retain 24/7 recordings forever

### DISCUSSION THREAD

**Discussion by dlong500** [blakeblackshear/frigate#6606] [Ask A Question] [answered]
Maybe I'm missing something simple, but if I want to manage the retention of 24/7 recordings manually (i.e. never delete) what should the configuration be?  Do I just need to put a ridiculously high number for `record > retain > days` or can this be disabled?

**Comment by blakeblackshear** [comment, marked as answer]
That would work. Frigate won't let the disk fill up, so if you are running out of storage, it will start deleting recordings.
